packages/engine/src/FeatureFlagsState.tsx

```diff
@@ -24,7 +24,7 @@ Ethereal Engine. All Rights Reserved.
 */
 
 import { featureFlagSettingPath } from '@etherealengine/common/src/schema.type.module'
-import { defineState, getMutableState, useHookstate } from '@etherealengine/hyperflux/functions/StateFunctions'
+import { defineState, getMutableState } from '@etherealengine/hyperflux/functions/StateFunctions'
 import { useFind } from '@etherealengine/spatial/src/common/functions/FeathersHooks'
 import { useEffect } from 'react'
 
```

Comment:
I think we need to completely get rid of this file.

Reply:
I pushed a commit to remove the scene preview camera flag, since the implementation to filter it from the categories state is creating a race conditions, and i dont think we need it anymore.

as for the hierarchy flag, i will need to talk to @MichaelEstes to see if there is a way we derive it from a feature flag

Reply:
I think this is good for now, we should add a deprecation notice to the state and call this PR good